target/scripts/build/packages.sh

```diff
@@ -116,106 +120,24 @@ function _install_dovecot() {
   apt-get "${QUIET}" --no-install-recommends install libxapian30
 }
 
-function _install_rspamd() {
-  _log 'trace' 'Adding Rspamd package signatures'
-  local DEB_FILE='/etc/apt/sources.list.d/rspamd.list'
-  local RSPAMD_PACKAGE_NAME
-
-  # We try getting the most recent version of Rspamd for aarch64 (from an official source, which
-  # is the backports repository). The version for aarch64 is 3.2; the most recent version for amd64
-  # that we get with the official PPA is 3.4.
-  #
-  # Not removing it later is fine as you have to explicitly opt into installing a backports package
-  # which is not something you could be doing by accident.
-  if [[ $(uname --machine) == 'aarch64' ]]; then
-    echo '# Official Rspamd PPA does not support aarch64, so we use the Bullseye backports' >"${DEB_FILE}"
-    echo 'deb [arch=arm64] http://deb.debian.org/debian bullseye-backports main' >>"${DEB_FILE}"
-    RSPAMD_PACKAGE_NAME='rspamd/bullseye-backports'
-  else
-    curl -sSfL https://rspamd.com/apt-stable/gpg.key | gpg --dearmor >/etc/apt/trusted.gpg.d/rspamd.gpg
-    local URL='[arch=amd64 signed-by=/etc/apt/trusted.gpg.d/rspamd.gpg] http://rspamd.com/apt-stable/ bullseye main'
-    echo "deb ${URL}" >"${DEB_FILE}"
-    echo "deb-src ${URL}" >>"${DEB_FILE}"
-    RSPAMD_PACKAGE_NAME='rspamd'
-  fi
-
-  _log 'debug' 'Installing Rspamd'
-  apt-get "${QUIET}" update
-  apt-get "${QUIET}" --no-install-recommends install "${RSPAMD_PACKAGE_NAME}" 'redis-server'
-}
-
-function _install_fail2ban() {
```

Comment:
I would keep fail2ban related code. The versions are now even, but in the future, they will drift away as happened in the past. For reason (bugfix) or another (missing feature), we then probably reintroduce it. So why not just keep it?

Reply:
I think @georglauterbach motivation was just a desire to trim out what he can if he's planning to continue as the primary maintainer of the project for his benefit? 🤷‍♂️ 

The method won't have as much relevance if moving to a different base image? If that happens it'd likely be a choice where the latest updates are available in faster release cycles. Depends how important it is to pull in the latest updates, but I understand both views here :)

I don't see any harm in leaving this method alone if it's not adding notable maintenance burden, the extra verbosity isn't an issue IMO.

Reply:
I would really like to get rid of this code, actually all the custom code that clutters `packages.sh` as of now. The same goes for Rspamd, where I also removed a lot of the custom code. It makes the whole script more verbose and harder to maintain. Using PPAs is fine as the package name stays consistent. From a maintenance PoV, removing the custom F2B code is very desirable.